src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -525,34 +525,34 @@ Autoplay can be changed in YouTube settings:
     <string name="revanced_preference_screen_action_buttons_title">Action buttons</string>
     <string name="revanced_preference_screen_action_buttons_summary">Hide or show action buttons under videos.</string>
 
-    <string name="revanced_hide_clip_button_title">Hide clip button</string>
+    <string name="revanced_hide_clip_button_title">Hide Clip button</string>
     <string name="revanced_hide_clip_button_summary_on">Clip button is hidden.</string>
     <string name="revanced_hide_clip_button_summary_off">Clip button is shown.</string>
-    <string name="revanced_hide_download_button_title">Hide download button</string>
+    <string name="revanced_hide_download_button_title">Hide Download button</string>
     <string name="revanced_hide_download_button_summary_on">Download button is hidden.</string>
     <string name="revanced_hide_download_button_summary_off">Download button is shown.</string>
     <string name="revanced_hide_like_dislike_button_title">Hide like and dislike buttons</string>
     <string name="revanced_hide_like_dislike_button_summary_on">Like and dislike buttons are hidden.</string>
     <string name="revanced_hide_like_dislike_button_summary_off">Like and dislike buttons are shown.</string>
-    <string name="revanced_hide_remix_button_title">Hide remix button</string>
+    <string name="revanced_hide_remix_button_title">Hide Remix button</string>
     <string name="revanced_hide_remix_button_summary_on">Remix button is hidden.</string>
     <string name="revanced_hide_remix_button_summary_off">Remix button is shown.</string>
-    <string name="revanced_hide_report_button_title">Hide report button</string>
+    <string name="revanced_hide_report_button_title">Hide Report button</string>
     <string name="revanced_hide_report_button_summary_on">Report button is hidden.</string>
     <string name="revanced_hide_report_button_summary_off">Report button is shown.</string>
-    <string name="revanced_hide_rewards_button_title">Hide rewards button</string>
+    <string name="revanced_hide_rewards_button_title">Hide Rewards button</string>
     <string name="revanced_hide_rewards_button_summary_on">Rewards button is hidden.</string>
     <string name="revanced_hide_rewards_button_summary_off">Rewards button is shown.</string>
-    <string name="revanced_hide_playlist_button_title">Hide save to playlist button</string>
-    <string name="revanced_hide_playlist_button_summary_on">Save to playlist button is hidden.</string>
-    <string name="revanced_hide_playlist_button_summary_off">Save to playlist button is shown.</string>
-    <string name="revanced_hide_share_button_title">Hide share button</string>
+    <string name="revanced_hide_playlist_button_title">"Hide 'Save to playlist' button"</string>
+    <string name="revanced_hide_playlist_button_summary_on">"'Save to playlist' button is hidden."</string>
+    <string name="revanced_hide_playlist_button_summary_off">"'Save to playlist' button is shown."</string>
```

Comment:
I'm not sure if it is correct that I put it in quotes. The button really only says "Save", so it should probably be unchanged, or the following:

```suggestion
    <string name="revanced_hide_playlist_button_title">Hide Save button</string>
    <string name="revanced_hide_playlist_button_summary_on">Save button is hidden.</string>
    <string name="revanced_hide_playlist_button_summary_off">Save button is shown.</string>
```

Reply:
In ReVanced:
https://github.com/ReVanced/revanced-patches/blob/b72fe87cf9b0a5d8105ead9b50f0c72cf8cc4c97/src/main/resources/addresources/values/strings.xml#L397

Reply:
Hmm, I don't think the way they did is consistent. Whatever you prefer

Reply:
Instead of using small quotes, it seems better to follow the method ReVanced adopted - capitalize only the first word

Reply:
Okay. So when are the quotes used?